README.md

```diff
@@ -1,9 +1,42 @@
 # rust-maven-plugin
-Build Rust Cargo crates within a Java Maven Project
+
+<img src="./artwork/logo_outline_text.svg" alt="rust-maven-plugin">
+
+Build Rust Cargo crates within a Java Maven Project with this plugin.
+
+The use case is to simplify the build process of
+[Rust JNI libs](https://crates.io/crates/jni) inside a Java
+[Maven](https://maven.apache.org/) project.
+
+# Features
+
```

Comment:
I think it might be worthwhile adding a line introducing the list because the list contains incomplete sentences..

something like:

The plugin provides the following features:

[changing the verb form for each item to calling/informing...etc and so on]

Reply:
Done.